### DISCUSSION TITLE
[Feature] Don't recommend other users

### DISCUSSION THREAD

**Discussion by vyPal** [immich-app/immich#12816] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I am hosting immich for a group of people, who dont all know each other, and they would prefer if their name and email was not recommended to other users when the other user decides to share an image to someone. 

I would either change this to a search based system, so that you could still search through users, or I'd make an option in the settings to sort of hide the user's identities from other users, or something like that, probably the first option though

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4408
